packages/eks-controller/templates/NOTES.txt

```diff
@@ -0,0 +1,4 @@
+You have deployed the Rancher EKS controller
+Version: {{ .Chart.AppVersion }}
```

Comment:
Is this normally Version or AppVersion?

Reply:
Its notes so can be anything you want, lot of charts just do chart name + release name. On that note, wonder if we should make a boilerplate notes for dev-charts which includes basic stuff, like where to file issues.